test/standalone_test.dart

```diff
@@ -108,9 +108,9 @@ void main() {
           d.file("bar$dotBat", anything),
           d.file("qux$dotBat", anything),
           d.dir("src", [
-            d.file("foo.dart.snapshot", anything),
-            d.file("bar.dart.snapshot", anything),
-            d.nothing("qux.dart.snapshot")
+            d.file("foo.snapshot", anything),
+            d.file("bar.snapshot", anything),
+            d.file("qux.snapshot", anything),
```

Comment:
Hmm, I guess using executable names would create multiple separate snapshots for an executable with multiple names. That's some amount of unnecessary overhead, and we *could* continue to generate snapshots based on filenames and have `testing.dart` just look up the snapshot path from the executable name. WDYT?

Reply:
What about having `pkg-compile-standalone` make a copy of the first snapshot if two executables use the same script?

Reply:
That works, although it does still add to the ultimate download size. I leave the ultimate decision up to you.